### PR TITLE
New simplify algorithm

### PR DESCRIPTION
**Pull request by jeromekelleher**
OK, this is as simple as simplify is going to get.  

The algorithm is actually pretty nice and concise now (and literally implemented in the Python code (see simplify-alg.py), so we could actually consider putting it back into the main text. The rest of you should make this call though; it's probably still a pretty obscure thing.

The description could do with another pass through from someone else, and a lookover to make sure I haven't made any mistakes.

I'll follow up with a few more commits and annotate my changes a bit later.

### CODE REVIEW

**Comment by molpopgen** [comment]
I quite like the idea of providing a Python implementation with the paper.   There are going to be folks who will prefer to code it up themselves rather than try to make the bridge to Python.  (In fact, I have an idea that may require doing so myself...)

**Comment by jeromekelleher** [comment]
> I quite like the idea of providing a Python implementation with the paper. There are going to be folks who will prefer to code it up themselves rather than try to make the bridge to Python. (In fact, I have an idea that may require doing so myself...)

I'm perfectly happy to include Python as SI. It's a good way of archiving something that definitely works...

**Comment by molpopgen** [comment]
> I'm perfectly happy to include Python as SI. It's a good way of archiving something that definitely works...

It is also a good way of making the algorithm notation concrete.  Some folks will just prefer to look at code.

File: forwards_paper.tex [hunk 1]
```diff
@@ -67,15 +72,15 @@
     and recording history as a \emph{succinct tree sequence} as introduced in the software package \msprime,
     on which neutral mutations can be quickly placed afterwards.
     We implement the method in two popular forwards-time simulation frameworks,
-    which speeds up large, whole-genome simulations by one to orders of magnitude.
+    which speeds up large, whole-genome simulations by one to two orders of magnitude.
```
Comment (petrelharp) [inline forwards_paper.tex:75]: heh, whoops

**Comment by petrelharp** [comment]
looks good; merging to make minor fixes

**Pull request merged**